internal/packages/assets.go

```diff
@@ -17,16 +17,35 @@ import (
 // AssetType represents the type of package asset.
 type AssetType string
 
+type assetTypeFolder struct {
+	Type       AssetType
+	folderName string
```

Comment:
Nit. Why is one public and the other one private?

Reply:
That is a leftover, I was trying first with another type of struct. But it was needed to updated more code.

Thanks for catching this! I'll rename it to `typeName` (I cannot set `type` since it is a reserved keyword).